kafka/src/main/scala/com.snowplowanalytics.snowplow.collectors.scalastream/TelemetryUtils.scala

```diff
@@ -0,0 +1,29 @@
+package com.snowplowanalytics.snowplow.collectors.scalastream
+
+import cats.effect.IO
+import org.http4s._
+import org.http4s.blaze.client.BlazeClientBuilder
+import org.typelevel.ci._
+import io.circe.parser
+
+object TelemetryUtils {
+
+  def getAzureSubscriptionId: IO[Option[String]] = {
+    val response = for {
+      client <- BlazeClientBuilder[IO].resource
+      request = Request[IO](
+        method  = Method.GET,
+        uri     = Uri.unsafeFromString("http://169.254.169.254/metadata/instance?api-version=2021-02-01"),
```

Comment:
What's the uri here? Isn't it region/instance specific? Is it globally accessible?